miden-crypto/src/merkle/smt/simple/mod.rs

```diff
@@ -348,7 +347,7 @@ impl<const DEPTH: u8> SparseMerkleTree<DEPTH> for SimpleSmt<DEPTH> {
     type Key = LeafIndex<DEPTH>;
     type Value = Word;
     type Leaf = Word;
-    type Opening = ValuePath;
+    type Opening = SparseValuePath;
```

Comment:
Did we decide against renaming `SparseValuePath` into `SimpleSmtProof` btw? Personally, I would still prefer that because what `SmtProof` is for `Smt`, `SparseValuePath` is for `SimpleSmt`. Just wondering though, and not necessary for this PR.

Reply:
I like it, I think it makes things more consistent. We then have:
- ValuePath for MerkleTree store,
- SmtProof for Smt
- SimpleSmtProof for SimpleSmt

I made the change here: @bobbinth holler if this is wrong.

Reply:
We could also rename `ValuePath` into `MerkleProof` to be consistent with the other ones - but that's for a different PR.

Reply:
`ValuePath` > `MerkleProof` PR: https://github.com/0xMiden/crypto/pull/483